address.go

```diff
@@ -65,6 +65,7 @@ type Address struct {
 // CreateAddressOptions is used to specify verification options for address
 // creation.
 type CreateAddressOptions struct {
+	// TODO: These should be booleans, not strings, like the other libs
```

Comment:
Could we include this fix as part of the next release?

Reply:
This would be a breaking change, we should save this for a major release